packages/commerce-sdk-react/src/components/ShopperExperience/Page/index.tsx

```diff
@@ -6,7 +6,7 @@
  */
 import React, {useContext, useEffect, useState} from 'react'
 import {Helmet} from 'react-helmet'
-import {Component as ComponentType, Region as RegionType, Page as PageType} from '../types'
+import {Component as ComponentType, Page as PageType} from '../types'
```

Comment:
```suggestion
import type {Component as ComponentType, Page as PageType} from '../types'
```
If you only import types from a file, you can use `import type` to tell TypeScript that the import can be removed when compiling.